pkgs/development/libraries/mesa/stubs.nix

```diff
@@ -1,50 +1,37 @@
 { stdenv
+, lib
 , libglvnd
-, mesa
 , OpenGL
 , testers
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "libGL";
-  inherit (if stdenv.hostPlatform.isDarwin then mesa else libglvnd) version;
+  version = if stdenv.hostPlatform.isDarwin then "4.1" else libglvnd.version;
   outputs = [ "out" "dev" ];
 
-  # On macOS, libglvnd is not supported, so we just use what mesa
-  # build. We need to also include OpenGL.framework, and some
-  # extra tricks to go along with. We add mesa’s libGLX to support
-  # the X extensions to OpenGL.
+  # On macOS, libglvnd is not supported, and mesa no longer builds (but it only
+  # provided a software renderer anyway). Provide the OpenGL framework as well
+  # as a pkg-config file for OpenGL.framework.
+  # GLX is not supported nor is OpenGL ES.
```

Comment:
Mesa’s GLX support does build again now thanks to your own PR, so this comment should probably be softened.